pxb/percona-xtrabackup-2.4-template.yml

```diff
@@ -210,6 +210,12 @@
             cd ${WORKSPACE}/storage/innobase/xtrabackup/test
         fi
 
+        if [[ ! -z ${bootstrap_url} ]]; then
+            rm -f bootstrap.sh
+            wget -O bootstrap.sh ${bootstrap_url}
+            chmod +x bootstrap.sh
+        fi
+
```

Comment:
what is the current directory where are you downloading bootstrap. You should replace existent one

Reply:
Line 202:
```
        if [ -d ${WORKSPACE}/xtrabackup ]
        then
            # Old (patch-based) directory layout

            cd ${WORKSPACE}/xtrabackup/test
        else
            # New (single binary) directory layout

            cd ${WORKSPACE}/storage/innobase/xtrabackup/test
        fi
```

Reply:
It changes current directory to `test` where bootstrap.sh is located